okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -192,7 +192,11 @@ public Level getLevel() {
         }
 
         logger.log("");
-        logger.log(buffer.readString(charset));
+        if (isPlainText(buffer.clone())) {
```

Comment:
This is a very expensive thing to do for every body when we're only checking the first 16 chars.